src/cli.ts

```diff
@@ -144,15 +143,21 @@ function runTranspile(files: string[], options: CliOptions) {
 }
 
 export function createCairoProject(filePath: string): void {
-  // create cairo_project.toml
-  const cairoProjectPath = path.join(path.dirname(filePath), 'cairo_project.toml');
-  const warplibRoot = path.join(PROJECT_ROOT, 'warplib');
+  const outputRoot = path.dirname(path.dirname(filePath));
+  const packageName = path.basename(outputRoot, '.sol').replace('-', '_');
+  const scarbConfigPath = path.join(outputRoot, 'Scarb.toml');
+  const warplib = path.join(BASE_PATH, 'warplib');
   outputFileSync(
-    cairoProjectPath,
+    scarbConfigPath,
     endent`
-      [crate_roots]
-      root = "."
-      warplib = "${warplibRoot}"
+    [package]
+    name = "${packageName}"
+    version = "1.0.0"
+
+    [dependencies]
+    warplib = { path = "${warplib}" }
```

Comment:
Could we use `PATH_TO_WARPLIB` here as well?

Reply:
They are not the same path. The one in `util` is for when you execute `yarn warplib` which you do from the root. The second one is done inside the bin folder (`bin/warp`). So you need to resolve first to get to the root.

Reply:
Can't we use absolute paths? Relative paths are always error prone

Reply:
It is an absolute path